src/webgl/light.js

```diff
@@ -169,6 +169,8 @@ p5.prototype.specularColor = function(v1, v2, v3) {
 
 /**
  * Creates a directional light with a color and a direction
+ * <br/>
```

Comment:
You can omit these `<br/>` elements and just leave a blank line and it will translate into a line break.

Reply:
Thanks for the information. Doing right now.

Reply:
Do I also need to add a workaround for increasing the maximum no. of lights in the documentation?

Means changing the array size in this file and building p5 from source https://github.com/processing/p5.js/blob/e11b4d617973a6cbeb789302ac093d26b8f2e489/src/webgl/shaders/lighting.glsl#L12

Reply:
No need to add the workaround to the documentation. Thank you!